code/modules/atmospherics/machinery/components/binary_devices/pressure_valve.dm

```diff
@@ -41,6 +43,14 @@
 		radio_connection = null
 	return ..()
 
+/obj/machinery/atmospherics/components/binary/pressure_valve/examine(mob/user)
+	. = ..()
+	. += "This device will let gas flow if the pressure of the gas in the input is [inverted ? "higher" : "lower"] than the pressure set in the interface."
+	if(inverted)
+		. += "The device settings can be restored if a multitool is used on it."
+	else
+		. += "The sensor's settings can be changed by using a multitool on the device."
+
```

Comment:
A message like this is in the discription of the valve too, you may not want to put this in examine, at least the let gas flow if the pressure is higher/lower part.

Reply:
The temp gate has this same issue

Reply:
Having some indication of the inversion is good, but it shouldn't double up.

Reply:
You should make this change for both the temp gate and the pressure gate, if it does in fact have the same issue as old me thought